website/pages/consul-on-kubernetes/index.jsx

```diff
@@ -1,14 +1,25 @@
 import Head from 'next/head'
-// import s from './style.module.css'
+import CKHero from 'components/ck-hero'
 
 export default function ConsulOnKubernetesPage() {
   return (
     <div>
       <Head>
         <title key="title">Consul on Kubernetes</title>
       </Head>
-      {/* hero */}
-
+      <CKHero
+        title="Consul on Kubernetes"
+        subtitle="A robust service mesh for discovering and securely connecting applications on Kubernetes."
+        ctas={[
+          { text: 'Get Started', url: '#TODO' },
+          { text: 'Try HCP Consul', url: '#TODO' },
+        ]}
+        media={{
+          type: 'image',
+          source: '/img/sample-video.png',
```

Comment:
Nice placeholder content,  really gives me the feel before we have the video ✨ 

Maybe pull this placeholder image outside of public, and import an image that's in an `image` directory here.  This is one of those scenarios where it would be easy to forget to delete it from public once we stop referencing it (but harder to forget about it if it's local).